js/scriptGame.js

```diff
@@ -0,0 +1,130 @@
+let isInventoryOpen = false;
+
+document.getElementsByClassName("inventorySlot slotArrow")[0].onclick = function() {
+    if(!isInventoryOpen){
+        moveInventory("-", 85, 0, 0);
+        isInventoryOpen = true;
+        document.getElementsByClassName("blackoutPanel")[0].classList.remove("hidden")
+    }
+    else if(isInventoryOpen){
```

Comment:
```suggestion 
     else if (isInventoryOpen) {
```
Let's have if's like this
```
if () {
    // code
} else {
   // code
}
```